fuel-core-interfaces/src/db.rs

```diff
@@ -792,33 +792,33 @@ pub mod helpers {
     }
 
     // Validator set. Used by relayer.
-    impl Storage<Address, (ValidatorStake, Option<Address>)> for DummyDb {
+    impl Storage<ValidatorId, (ValidatorStake, Option<ConsensusId>)> for DummyDb {
         type Error = crate::db::KvStoreError;
 
         fn insert(
             &mut self,
-            key: &Address,
-            value: &(ValidatorStake, Option<Address>),
-        ) -> Result<Option<(ValidatorStake, Option<Address>)>, Self::Error> {
+            key: &ValidatorId,
+            value: &(ValidatorStake, Option<ConsensusId>),
+        ) -> Result<Option<(ValidatorStake, Option<ConsensusId>)>, Self::Error> {
             Ok(self.data.lock().validators.insert(*key, *value))
         }
 
         fn remove(
             &mut self,
-            _key: &Address,
-        ) -> Result<Option<(ValidatorStake, Option<Address>)>, Self::Error> {
+            _key: &ValidatorId,
+        ) -> Result<Option<(ValidatorStake, Option<ConsensusId>)>, Self::Error> {
             unreachable!()
```

Comment:
Unrelated to this PR, but how is that unreachable since the inner counterpart is a hashmap that trivially implements `remove`? If we really want to block keys removal, we should return an `Err` instead - and not panic.

Reply:
It is DummyDB used for testing. I created task for it: https://github.com/FuelLabs/fuel-core/issues/431